keras/preprocessing/image.py

```diff
@@ -364,6 +418,9 @@ class ImageDataGenerator(object):
         rescale: rescaling factor. If None or 0, no rescaling is applied,
             otherwise we multiply the data by the value provided
             (before applying any other transformation).
+        hsv_augmentation: parameters for HSV data augmentation
+                          (hue shift, saturation scale, saturation shift,
+                          value scale, and value shift)
```

Comment:
Is a tuple of 5 values a good API? Also you don't provide much information about expect values and their meaning. e.g. what is "value shift"? A range? Fractional? From reading this docstring I have no idea what would be some good values to pass.